models/migrations/migrations.go

```diff
@@ -889,3 +891,13 @@ func modifyColumn(x *xorm.Engine, tableName string, col *schemas.Column) error {
 	}
 	return nil
 }
+
+// addColumn adds new column to the specified table
+func addColumn(x *xorm.Engine, tableName string, col *schemas.Column) error {
```

Comment:
But this will not check if the column is exist.

Reply:
If column exists xorm throw an error. I think, is overkill for checking there.